sanic/static.py

```diff
@@ -103,7 +104,7 @@ def register(app, uri, file_or_directory, pattern,
                     if isinstance(stream_large_files, int):
                         threshold = stream_large_files
                     else:
-                        threshold = 1024*1000
+                        threshold = 1024 * 1000
```

Comment:
I found this one while acting on https://github.com/channelcat/sanic/pull/1005#discussion_r148696110